AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -444,6 +444,7 @@ file.fm,files.fm#%#//scriptlet("set-constant", "canRunAds", "true")
 moregameslike.com#%#//scriptlet("abort-on-property-read", "adbh")
 ||googleads.g.doubleclick.net/pagead/$redirect=nooptext,important,domain=maxedtech.com
 ||atc.studybreakmedia.com/modules/abb^
+nickles.de#%#//scriptlet('set-constant', 'Nickles.Consent.show', 'noopFunc')
```

Comment:
I think that it would be better to move this rule to  `annoyances.txt`, because it's not an anti-adblock script.